### ISSUE TITLE
Database location is ignored

### ISSUE THREAD

**Issue by efreet7** [blakeblackshear/frigate-hass-addons#19]
configuration.yaml
`homeassistant: `
  ` media_dirs:`
    `  media: /data/media`

frigate.yml
`database:`
  ` /media/frigate/clips/frigate.db`

Host volume to path in homeassistant container
`/mnt/data -> /data`

`/mnt/data` is a mounted hardrive

Instead of storing the database and recordings at full path of `/mnt/data/media/frigate/clips/frigate.db`. It puts it at default location of `/usr/share/hassio/media/frigate/clips`

**Comment by blakeblackshear** [comment]
I don’t think this has anything to do with frigate or this addon. Hassos supervisor manages all of storage and volumes.
